src/UI/Components/Diagram/Canvas.tsx

```diff
@@ -49,27 +63,42 @@ const Canvas = ({
   };
 
   const handleDeploy = async () => {
-    const mapToArray = Array.from(instancesToSend, (instance) => instance[1]); //only value, the id is stored in the object anyway
-    const topServicesNames = services.map((service) => service.name);
-    const topInstances = mapToArray.filter((instance) =>
-      topServicesNames.includes(instance.service_entity),
-    );
-    const embeddedInstances = mapToArray.filter(
-      (instance) => !topServicesNames.includes(instance.service_entity),
-    );
+    try {
+      const response = await fetch(`${baseUrl}/lsm/v2/order`, {
```

Comment:
I think we can make a ticket, to at least extract all API interactions into a neat folder. If we want to replace it at a future stage with a better way, it will be easier to handle it.

Reply:
Created separate ticket for that